src/libaktualizr/primary/CMakeLists.txt

```diff
@@ -12,9 +12,12 @@ set(HEADERS aktualizr.h
 
 
 add_library(primary OBJECT ${SOURCES})
+target_include_directories(primary PUBLIC ${PROJECT_SOURCE_DIR}/src/virtual_secondary)
 
 add_aktualizr_test(NAME aktualizr SOURCES aktualizr_test.cc PROJECT_WORKING_DIRECTORY ARGS ${PROJECT_BINARY_DIR}/uptane_repos)
 add_dependencies(t_aktualizr uptane_repo_full_no_correlation_id)
+target_include_directories(t_aktualizr PUBLIC ${PROJECT_SOURCE_DIR}/src/virtual_secondary)
```

Comment:
cleaner CMake way: don't add the include directories in the library consumers manually but do it for the `virtual-secondary` library and link with them